alf/algorithms/algorithm.py

```diff
@@ -1492,7 +1492,13 @@ def _prepare_experience_data(self,
             experience, experience_spec)
         experience = alf.data_structures.add_batch_info(
             experience, batch_info, self._replay_buffer)
+        untransformed_time_step = None
+        if hasattr(experience, 'time_step'):
+            untransformed_time_step = experience.time_step
```

Comment:
Perhaps it's better to add a new DataTransformer whose sole function is to save the untransformed_time_step. untransformed_time_step is not used in most cases and sometime the time_step use a lot of memory.

Reply:
> Perhaps it's better to add a new DataTransformer whose sole function is to save the untransformed_time_step. untransformed_time_step is not used in most cases and sometime the time_step use a lot of memory.

updated